### ISSUE TITLE
Bot not working again

### ISSUE THREAD

**Issue by gfys** [finndev/PokeBuddy#1046]
I've downloaded the most recent version of the bot, but about 4 days later its stopped recognizing pokestops again.

**Comment by AlexCtasy** [comment]
Stop using pokebuddy, is getting auto ban.

**Comment by gfys** [comment]
im not

**Comment by gfys** [comment]
im using the rocket api at 25 km/hr

**Comment by ZzYaPoO** [comment]
i agree gfys mentioned above., i creATED NEW ACCOUNT for testing, i tried bot from 0- 10, without login to mobile device. rocket api at 25km/h. already 11lv. bot time is Malaysia time from 1800-1900, so far is ok....
i dont think is autoban.
